Source/Core/Core/PowerPC/Jit64/Jit_LoadStorePaired.cpp

```diff
@@ -302,16 +354,24 @@ void Jit64::psq_lXX(UGeckoInstruction inst)
   // In memcheck mode, don't update the address until the exception check
   if (update && !jo.memcheck)
     MOV(32, gpr.R(a), R(RSCRATCH_EXTRA));
-  MOV(32, R(RSCRATCH2), Imm32(0x3F07));
 
-  // Get the high part of the GQR register
-  OpArg gqr = PPCSTATE(spr[SPR_GQR0 + i]);
-  gqr.AddMemOffset(2);
+  if (gqrIsConstant)
+  {
+    GenQuantizedLoad(w == 1, static_cast<EQuantizeType>(gqrValue & 0x7), (gqrValue & 0x3F00) >> 8);
```

Comment:
Yep. There's still a lot of improvement possible here, but the `MOV_sum` work in the regcache stuff makes all of this _so_ much easier to work with. I definitely want to follow up with further improvements.